gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -89,6 +89,11 @@ grant {
   permission java.lang.reflect.ReflectPermission "newProxyInPackage.net.bytebuddy.description.type";
   permission java.lang.reflect.ReflectPermission "newProxyInPackage.net.bytebuddy.dynamic.loading";
   permission java.lang.reflect.ReflectPermission "newProxyInPackage.net.bytebuddy.utility";
+  // Allow execution of 'java.exe' anywhere on the system for Windows
+  permission java.io.FilePermission "C:\Users\mark\.jdks\openjdk-19.0.1\bin\java.exe", "execute";
```

Comment:
What do we need to change this to make it mergable?

Reply:
That's the stuff that doesn't work that I told you about, so I don't know that it matters either way from the perspective that tests wont pass with the security manager on. I believe it's bytebuddy from mockito that needs to be executed on java.exe, but nothing I've tried seems to give it. It always fails to say it needs that permission. I tried a variety of things that should give it, and none of them worked.

Reply:
I updated to remove those additions, but of course, it still needs a working version of those additions.